stata_kernel/ado/_StataKernelHead.ado

```diff
@@ -15,7 +15,7 @@ program _StataKernelHead
         if ( _rc ) error 198
         local n = `n1'
     }
-    else local n = 10
+    else local n = `N_default'
```

Comment:
I don't think this works bc uppercase is a shortcut. Should be `n_default`

Reply:
oh lol yeah